src/modules/Portal/components/Search/Results/List/ResultItem/components/Highlight/presenter.jsx

```diff
@@ -0,0 +1,52 @@
+/*
+  *
+  * Copyright 2017 Observational Health Data Sciences and Informatics
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *     http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  *
+  * Company: Odysseus Data Services, Inc.
+  * Product Owner/Architecture: Gregory Klebanov
+  * Authors: Pavel Grafkin, Alexander Saltykov, Vitaly Koulakov, Anton Gackovka, Alexandr Ryabokon, Mikhail Mironov
+  * Created: Monday, March 19, 2018 11:24 AM
+  *
+  */
+
+import React from 'react';
+import BEMHelper from 'services/BemHelper';
+
+import './style.scss';
+
+const detectorRegexp = /(\[b\][^\[]*\[\/b\])/g;
```

Comment:
Why two regexps?

Reply:
cause one seeks [b] in the start of the string, and [/b] in the end

Reply:
Why do job twice? Use match